test/apiv2/20-containers.at

```diff
@@ -63,7 +63,7 @@ cid=$(jq -r '.Id' <<<"$output")
 # Prior to the fix in #6835, this would fail 500 "args must not be empty"
 t POST   libpod/containers/${cid}/start '' 204
 # Container should exit almost immediately. Wait for it, confirm successful run
-t POST   libpod/containers/${cid}/wait  '' 200 '0'
+t POST   libpod/containers/${cid}/wait?condition=stopped&condition=exited  '' 200 '0'
```

Comment:
please put the URI into quotes
`t POST   "libpod/containers/${cid}/wait?condition=stopped&condition=exited"  '' 200 '0'`

Reply:
Fixed in https://github.com/containers/podman/pull/9414/commits/3e168b19f23d3e9e4e9f25d0368b4d607968abb4.